### ISSUE TITLE
`Undefined node "type"` error will occur when I use webpack in production mode

### ISSUE THREAD

**Issue by takashiki** [glayzzle/php-parser#173]
When I pack code in development mode, it will work properly. But if I use production mode, error like bellow will occur everytime.

Input is: `['a' => 1]`:

```
Uncaught Error: Undefined node "string"
    at main.js:210
    at n.read_scalar (main.js:174)
    at n.read_expr_item (main.js:144)
    at n.read_expr (main.js:144)
    at n.read_array_pair_list (main.js:129)
    at n.read_array (main.js:129)
    at n.read_scalar (main.js:174)
    at n.read_expr_item (main.js:144)
    at n.read_expr (main.js:144)
    at n.read_statement (main.js:179)
```

I've tried both 2.x & 3.x-alpha, same result got.

**Comment by ichiriac** [comment]
Hi @takashiki,

With what tool do you pack the code (and configuration options). I'm actually using a packed version released here https://github.com/glayzzle/php-parser/blob/master/dist/php-parser.min.js and it works.

Here what I do for the compression : https://github.com/glayzzle/php-parser/blob/master/webpack.config.js

The error message you have means that this folder : https://github.com/glayzzle/php-parser/tree/master/src/ast is not correctly included into the source but as the ast.js requires explicitly every file there is no reason to not be bundled correctly.

**Comment by takashiki** [comment]
I tested some times and find out when I add `UglifyJsPlugin` part, that error will disappear.

**Comment by ichiriac** [comment]
Thanks @takashiki, I will try to figure out what does not work in production mode with babel & their defaut JS compressor, meanwhile use the UglifyjsPlugin as it's our default building process.

**Comment by takashiki** [comment]
Without config bellow in `UglifyOptions`, this error will occur too.
```
compress: {
    keep_fnames: true
}
```

**Comment by ichiriac** [comment]
Hi @takashiki, the problem should be resolved by using version `3.0.0-prerelease.2`
